manifests/osfamily/debian.pp

```diff
@@ -7,6 +7,14 @@
     key        => {
       'id'     => '47B320EB4C7C375AA9DAE1A01054B7A24BD6EC30',
       'server' => 'pgp.mit.edu',
-    }
+    },
+    notify     => Notify['pc1_repo_force'],
+  }
+
+  # apt_update doesn't inherit the future class dependency, so it
+  # can wait until the end of the run to exec. Force it to happen now.
+  notify { 'pc1_repo_force':
```

Comment:
The metadata specifies puppetlabs-apt 2.0.1. I'm not exactly sure what acceptance is installing though, I'll double check.